javascript/README.md

```diff
@@ -88,6 +88,21 @@ var length = items.length;
 var name = 'foo';
 ````
 
+## Don't have unused variables/expressions
+```javascript
+// Bad: Many of these variables/expressions are useless
+{} // object - not used
+var newArray = []; // newArray - not used
+['1', '2', '3', '4'].forEach(function(num, index) { // index - not used
```

Comment:
@jeremywiebe The `2` indicates that a rule violation will surface an `error` instead of just a `warning` (exit code of 1).

http://eslint.org/docs/user-guide/configuring#configuring-rules